extending-valtimo/document/assigning-a-user.md

```diff
@@ -0,0 +1,21 @@
+# Assigning a user
+
+It is possible to assign a user to a document. There are two methods available
```

Comment:
Missing a short introduction here. Something like https://github.com/valtimo-platform/valtimo-documentation/blob/main/using-valtimo/document/create-document-definition.md
Maybe you can copy the first part of that introduction? So it becomes something like:

A document definition is a JSON document that defines the structure that document has, as well as restrict the allowed values for each property through validation rules. This page shows how to assign and unassign users from and to a case.

## Assigning a user
There are two ways to assign a user to a case:
...

## Unassigning a user
...